src/ext/logging.c

```diff
@@ -2,13 +2,19 @@
 
 #include <php.h>
 
-void _ddtrace_log_errf(TSRMLS_FC const char *format, ...) {
+void ddtrace_log_err(char *message) {
```

Comment:
I would put this in the header as `inline` (not static inline) and then here in the .c file put the `extern inline` declaration. I've done this is some places already; search for `extern inline` for examples.

Reply:
Excellent feedback. Updated! :)